packages/bpk-component-breakpoint/src/BpkBreakpoint-test.tsx

```diff
@@ -53,21 +53,8 @@ describe('BpkBreakpoint', () => {
   });
 
   describe('SSR mode', () => {
-    let windowSpy: any;
-
     beforeEach(() => {
-      // The following spy/mock removes the createElement function to fool the BpkBreakpoint into
```

Comment:
This was left over from a previous way to detect SSR so is not necessary to mock the window anymore, the current implementation relies on the execution of a `useEffect` hook being executed as evidence that it is running on the client rather than the presence of the window object.